examples/SimpleMontage.py

```diff
@@ -0,0 +1,106 @@
+# ==========================================================================
+#
+#   Copyright NumFOCUS
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#          http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ==========================================================================*/
+import sys
+import os
+import itkConfig
+itkConfig.LazyLoading = False
+import itk
+
+itk.auto_progress(2)
+
+if len(sys.argv) < 4:
+  print(f"Usage: {sys.argv[0]} <directoryWithInputData> <outputDirectory> <outputFilename>")
+  sys.exit(1)
+
+inputPath = sys.argv[1]
+outputPath = sys.argv[2]
+outFile = sys.argv[3]
+
+if not inputPath.endswith('/') and not inputPath.endswith('\\'):
+  inputPath += '/'
+
+if not outputPath.endswith('/') and not outputPath.endswith('\\'):
+  outputPath += '/'
+
+os.chdir(outputPath)
+outFile = os.path.abspath(outFile)
```

Comment:
Once a `Path` instance, we can get this [with `out_file.resolve()`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve).